apps/web/app/(app)/mail/page.tsx

```
@@ -20,65 +20,81 @@ export default function Mail({
     [mutate],
   );
 
-  // store `refetch` in the atom so we can refresh the list upon archive via command k
-  // TODO is this the best way to do this?
+  // Set up the refetch function in the atom store
   const setRefetchEmailList = useSetAtom(refetchEmailListAtom);
   useEffect(() => {
     setRefetchEmailList({ refetch });
   }, [refetch, setRefetchEmailList]);
 
+  const handleLoadMore = useCallback(() => {
```

Biome

[error] This hook does not specify all of its dependencies: setSize

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)